torch/nested/creation.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import numbers
+import warnings
+
+
+def nested_tensor(
+    data,
+    dtype=None,
+    device=None,
+    requires_grad=False,
+    pin_memory=False,
+    channels_last=False,
```

Comment:
this is not symmetric with how factory functions work. I'd prefer if this API lined up with our preexisting factory functions

Reply:
Also it's kind of weird that you have this flag but it has to be False. Why not just delete it for now (and at it back later when memory format support comes online?)

Reply:
Updating it right now. Will orient myself on functions such as ones etc.